ocp_resources/resource.py

```diff
@@ -654,13 +655,21 @@ def update_replace(self, resource_dict):
         self.api.replace(body=resource_dict, name=self.name, namespace=self.namespace)
 
     @staticmethod
-    def _retry_cluster_exceptions(func):
+    def _retry_cluster_exceptions(
+        func,
+        **kwargs,
+    ):
+        exceptions_dict = DEFAULT_CLUSTER_RETRY_EXCEPTIONS
+        if "exceptions_dict" in kwargs:
```

Comment:
`exceptions_dict` should be in arg, kwargs if for the func kwargs to pass to the sampler.

Reply:
Ok, that makes sense. I will update this PR soon.